test/unit/specs/users/cd-tickets.spec.js

```diff
@@ -42,17 +42,15 @@ describe('Events/Tickets list component', () => {
         const vm = vueUnitHelper(TicketListWithMocks);
 
         // ACT
-        vm.loadCurrentUser();
+        await vm.loadCurrentUser();
 
         // ASSERT
-        requestAnimationFrame(() => {
-          expect(vm.currentUser).to.deep.equal(mockUser);
-          done();
-        });
+        expect(vm.currentUser).to.deep.equal(mockUser);
+        return Promise.resolve();
```

Comment:
No need to `return Promise.resolve();`. async functions implicitly do this.

Reply:
You were right btw, dunno what went wrong first time :|

Reply:
/me does my _I was right_ dance. 😝